src/modules/service.js

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable import/extensions */
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] Let's do the same here. Kindly enable linters to help us have a product that is error-free.